### PR TITLE
add mismatch to load weight

### PR DESCRIPTION
**Pull request by cylee81**
This PR adds the shape mismatch option to tf.keras load_weight function which ignores weights that are different with graph. This function is originally from: https://github.com/keras-team/keras/blob/ed07472bc5fc985982db355135d37059a1f887a9/keras/engine/saving.py#L1202

### CODE REVIEW

**Review by tanzhenyu — COMMENTED** [review]
Thanks for adding it! While it LGTM because this is mostly a duplicate from external Keras, would you mind adding a test for it? (under saving_test)

**Comment by gbaned** [comment]
@cylee81 Did you get a chance to look on reviewer comments? Please let us know on the update. Thanks!

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by cylee81** [comment]
Hi! I am not quite sure where the saving_test.py located.

**Comment by tanzhenyu** [comment]
anything under [this](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python/keras/saving) is fine.

**Comment by cylee81** [comment]
Hi I already finished the testing function (under tensorflow/python/keras/saving/save_test.py). Please check it. Thank you!

**Comment by gbaned** [comment]
@cylee81 Could you please resolve the conflicts? Thanks!

**Comment by cylee81** [comment]
Hi @gbaned, I fixed it. Please check it. Thanks.

**Review by tanzhenyu — DISMISSED** [review]

**Comment by cylee81** [comment]
Hi @gbaned @tanzhenyu , I fixed the pylint issue. Please check. Thank you.

**Review by tanzhenyu — APPROVED** [review]

**Comment by gbaned** [comment]
@cylee81 Could you please resolve the conflicts? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by cylee81** [comment]
I believe this was made redundant by c52ee53#diff-4ee308ea180d49ae81691348531a2b6d . Closing this.
